web/src/cypress/integration/profile.ts

```diff
@@ -37,15 +37,32 @@ function testProfile(screen: ScreenFormat) {
       const type = c.pickone(['SMS', 'VOICE'])
 
       cy.pageFab('Contact')
-      cy.get('input[name=name]').type(name)
-      cy.get('input[name=type]').selectByLabel(type)
-      cy.get('input[name=value]').type(value)
-      cy.get('button[type=submit]').click()
+      cy.get(`[data-cy='create-form']`)
+        .get('input[name=name]')
+        .type(name)
+        .get('input[name=type]')
+        .selectByLabel(type)
+        .get('input[name=value]')
+        .type(value)
+        .get('button[type=submit]')
+        .click()
+      cy.get(`[data-cy='verify-form']`)
+        .find('button[type=button]')
```

Comment:
This can be replaced/merged with the below line as `.contains('button[type=button]', 'Cancel')`